packages/app-mobile/components/NoteEditor/CodeMirror.ts

```diff
@@ -177,12 +66,45 @@ export function initCodeMirror(parentElement: any, initialText: string, theme: a
 		}, doItNow ? 0 : 1000);
 	}
 
+	function notifyDocChanged(viewUpdate: ViewUpdate) {
```

Comment:
It's a small thing but for new code please create only arrow functions `const notifyDocChanged = (.....) => {`

This is to avoid having a "this" in the function body.

Reply:
I've changed this!

I'm not sure this applies to nested functions, however:
```js
export function initCodeMirror(
	parentElement: any, initialText: string, settings: EditorSettings
): EditorControl {

	... snipped ...

	console.log("Extern this:", this);
	(function() {
		console.warn("this in anonymous:", this);
	})();

	function foo() {
		console.warn("this in foo:", this);
	}
	foo();
	const bar = () => {
		console.log("this in bar:", this);
	}
	bar();
```
![Console: extern this is not undefined, local this is.](https://user-images.githubusercontent.com/46334387/175827263-0f405faa-442a-405a-b730-d2efd2bd27ae.png)